2021-09-29 17:21:30,482 INFO    MainThread:35805 [wandb_setup.py:_flush():69] setting env: {}
2021-09-29 17:21:30,482 INFO    MainThread:35805 [wandb_setup.py:_flush():69] setting login settings: {}
2021-09-29 17:21:30,482 INFO    MainThread:35805 [wandb_init.py:_log_setup():348] Logging user logs to /home/pholur/Situation_Modeling/wandb/run-20210929_172130-182x3318/logs/debug.log
2021-09-29 17:21:30,482 INFO    MainThread:35805 [wandb_init.py:_log_setup():349] Logging internal logs to /home/pholur/Situation_Modeling/wandb/run-20210929_172130-182x3318/logs/debug-internal.log
2021-09-29 17:21:30,483 INFO    MainThread:35805 [wandb_init.py:init():381] calling init triggers
2021-09-29 17:21:30,483 INFO    MainThread:35805 [wandb_init.py:init():388] wandb.init called with sweep_config: {}
config: {}
2021-09-29 17:21:30,483 INFO    MainThread:35805 [wandb_init.py:init():430] starting backend
2021-09-29 17:21:30,483 INFO    MainThread:35805 [backend.py:_multiprocessing_setup():72] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2021-09-29 17:21:30,491 INFO    MainThread:35805 [backend.py:ensure_launched():135] starting backend process...
2021-09-29 17:21:30,555 INFO    MainThread:35805 [backend.py:ensure_launched():140] started backend process with pid: 39964
2021-09-29 17:21:30,566 INFO    MainThread:35805 [wandb_init.py:init():435] backend started and connected
2021-09-29 17:21:30,569 INFO    MainThread:35805 [wandb_init.py:init():494] updated telemetry
2021-09-29 17:21:30,570 INFO    MainThread:35805 [wandb_init.py:init():517] communicating current version
2021-09-29 17:21:31,843 INFO    MainThread:35805 [wandb_init.py:init():522] got version response 
2021-09-29 17:21:31,843 INFO    MainThread:35805 [wandb_init.py:init():530] communicating run to backend with 30 second timeout
2021-09-29 17:21:32,205 INFO    MainThread:35805 [wandb_init.py:init():557] starting run threads in backend
2021-09-29 17:21:37,212 INFO    MainThread:35805 [wandb_run.py:_console_start():1605] atexit reg
2021-09-29 17:21:37,214 INFO    MainThread:35805 [wandb_run.py:_redirect():1479] redirect: SettingsConsole.REDIRECT
2021-09-29 17:21:37,214 INFO    MainThread:35805 [wandb_run.py:_redirect():1484] Redirecting console.
2021-09-29 17:21:37,222 INFO    MainThread:35805 [wandb_run.py:_redirect():1540] Redirects installed.
2021-09-29 17:21:37,222 INFO    MainThread:35805 [wandb_init.py:init():582] run started, returning control to user process
2021-09-29 17:22:20,087 INFO    MainThread:35805 [wandb_run.py:_atexit_cleanup():1575] got exitcode: 0
2021-09-29 17:22:20,107 INFO    MainThread:35805 [wandb_run.py:_restore():1547] restore
2021-09-29 17:22:22,594 INFO    MainThread:35805 [wandb_run.py:_wait_for_finish():1697] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 736
  total_bytes: 736
}

2021-09-29 17:22:23,524 INFO    MainThread:35805 [wandb_run.py:_wait_for_finish():1697] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 736
  total_bytes: 736
}

2021-09-29 17:22:23,662 INFO    MainThread:35805 [wandb_run.py:_wait_for_finish():1697] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 736
  total_bytes: 736
}

2021-09-29 17:22:24,337 INFO    MainThread:35805 [wandb_run.py:_wait_for_finish():1697] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 736
  total_bytes: 21467
}

2021-09-29 17:22:24,440 INFO    MainThread:35805 [wandb_run.py:_wait_for_finish():1697] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 736
  total_bytes: 24551
}

2021-09-29 17:22:24,546 INFO    MainThread:35805 [wandb_run.py:_wait_for_finish():1697] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 3411
  total_bytes: 24551
}

2021-09-29 17:22:24,648 INFO    MainThread:35805 [wandb_run.py:_wait_for_finish():1697] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 18606
  total_bytes: 24551
}

2021-09-29 17:22:24,751 INFO    MainThread:35805 [wandb_run.py:_wait_for_finish():1697] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 18606
  total_bytes: 24551
}

2021-09-29 17:22:24,853 INFO    MainThread:35805 [wandb_run.py:_wait_for_finish():1697] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 18606
  total_bytes: 24551
}

2021-09-29 17:22:24,956 INFO    MainThread:35805 [wandb_run.py:_wait_for_finish():1697] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 24551
  total_bytes: 24551
}

2021-09-29 17:22:25,059 INFO    MainThread:35805 [wandb_run.py:_wait_for_finish():1697] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 24551
  total_bytes: 24551
}

2021-09-29 17:22:25,162 INFO    MainThread:35805 [wandb_run.py:_wait_for_finish():1697] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 24551
  total_bytes: 24551
}

2021-09-29 17:22:25,266 INFO    MainThread:35805 [wandb_run.py:_wait_for_finish():1697] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 24551
  total_bytes: 24551
}

2021-09-29 17:22:25,388 INFO    MainThread:35805 [wandb_run.py:_wait_for_finish():1697] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 24551
  total_bytes: 24551
}

2021-09-29 17:22:25,672 INFO    MainThread:35805 [wandb_run.py:_wait_for_finish():1697] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 24551
  total_bytes: 24551
}
local_info {
}

2021-09-29 17:22:27,368 INFO    MainThread:35805 [wandb_run.py:_show_files():1947] logging synced files
